scipy/stats/tests/test_hypotests.py

```diff
@@ -616,6 +619,43 @@ def test_gh_2118(self, x, y, alternative, expected):
                            method="asymptotic")
         assert_allclose(res, expected, rtol=1e-12)
 
+    def teardown(self):
```

Comment:
Is this right? I have had trouble finding official documentation for `setup` and `teardown` methods in `pytest` classes. Everything seems to be written about fixtures in general.

Reply:
> Is this right? I have had trouble finding official documentation for `setup` and `teardown` methods in `pytest` classes. Everything seems to be written about fixtures in general.

Yes, I think so. I have seen a lot of tests use the setup method in normal pytest classes. I also confirmed that pytest runs the teardown method after running all the tests. Although, I too can't find that stated explicitly on their doc pages.